README.adoc

```diff
@@ -180,7 +176,7 @@ Setting this to *true* prevents the TypeScript compiler from creating actual Jav
 
 target:: Specifies which JavaScript version your TypeScript code should be converted to. By default, TypeScript targets older versions of JavaScript for maximum compatibility.
 +
-Setting this to *"ES2023"* allows you to use modern JavaScript features that Foundry supports.
+Since Foundy itself uses very modern features, setting this to *"ES2023"* aligns with Foundry.
```

Comment:
So if you set `target` to a lower version you can still _use_ those features as long as your `lib` setting allows it.

It just so happens that `lib`, if unset, equals `target`. If you wanted you could set `"lib": "esnext"` and use `esnext` features while setting `"target": "es3"` to downlevel to es3.

This is part of what I mean by TypeScript being a bloated tool in that it tries to do everything for you.